MapboxCoreNavigation/RouteOptions.swift

```diff
@@ -24,6 +24,11 @@ extension RouteOptions {
         return copy
     }
     
+//    init(options: RouteOptions) {
```

Comment:
Is the idea to implement copying functionality? I think it would be better to keep pushing on mapbox/MapboxDirections.swift#388 with an eye towards turning RouteOptions into a struct, but it would be OK to continue to conform to NSCopying.